CONTRIBUTING.md

```diff
@@ -22,6 +22,7 @@ Jib comes as 3 components:
   - `jib-core`: a library
   - `jib-maven-plugin`: a Maven plugin that uses `jib-core`
   - `jib-gradle-plugin`: a Gradle plugin that uses `jib-core`
+  - `jib-plugins-common`: a library with helpers for plugin builders
```

Comment:
I think we can move this after `jib-core` and say the plugins use both `jib-core` and `jib-plugins-common`.